.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,95 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: ["main"]
+    # Publish semver tags as releases.
+    tags: ["v*.*.*"]
+  pull_request:
+    branches: ["main"]
+
```

Comment:
Consider adding the following snippet to free up runners when pushing on the same branch. Note: we should also add it to https://github.com/clicepfl/clic-website-v2/blob/main/.github/workflows/conventional-commits.yml
```suggestion

# abort running jobs if newer version is detected on same branch
concurrency:
  group: ${{ github.head_ref || github.ref }}
  cancel-in-progress: true

```